cmake/modules/extensions.cmake

```diff
@@ -1289,6 +1289,15 @@ function(zephyr_linker_sources location)
   endforeach()
 endfunction(zephyr_linker_sources)
 
+# Helper function for conditionally calling zephyr_code_relocate() when
+# a specific Kconfig symbol is enabled.
+# See zephyr_code_relocate() description for supported arguments.
+#
+macro(zephyr_code_relocate_ifdef feature_toggle)
```

Comment:
you write `Helper function` above, but this is a macro.
https://github.com/zephyrproject-rtos/zephyr/blob/fb223ce928a5ac3d77f5b28e82987af21abb84f7/cmake/modules/extensions.cmake#L1292

Please align text and implementation.

Reply:
Thanks, I aligned the lines in the description and changed s/function/macro.